Packs/SecurityScorecard/Integrations/SecurityScorecard/SecurityScorecard.py

```diff
@@ -0,0 +1,1078 @@
+        (i.e. xxxxxxxxxxxxxxxxxxxxxxxxx,yyyyyyyyyyyyyyyyyyyyy) to monitor with the alert.
+        The alert will be triggered once any one of the companies within the specified Portfolio
+        meets the set threshold This argument is require if the `target` argument is not specified.
+
+    :return:
+        A ``CommandResults`` object that is passed to ``return_results``
+    :rtype: ``CommandResults``
+    """
+
+    email = demisto.params().get('username').get("identifier")
+    change_direction = str(args.get('change_direction'))
+    threshold = arg_to_number(args.get('threshold'))  # type: ignore
+    score_types = argToList(args.get('score_types'))
+    target_arg = argToList(args.get('target'))
+    portfolios = argToList(args.get('portfolios'))
+
+    # Only one argument between portfolios and target should be defined
+    # Return error if neither of them is defined or if both are defined
+    # Else choose the one that is defined and use it as the target
+    if portfolios and target_arg:
+        raise DemistoException("""Both 'portfolio' and 'target' argument have been set.
+        Please remove one of them and try again.""")
+    elif target_arg and not portfolios:
+        target = target_arg
+    elif portfolios and not target_arg:
+        target = portfolios
+    else:
+        raise DemistoException("Either 'portfolio' or 'target' argument has to be speficied")
+
+    demisto.debug("Attempting to create alert with body {0}".format(args))
+    response = client.create_score_threshold_alert(
+        email=email,
+        change_direction=change_direction,
+        threshold=threshold,  # type: ignore
+        score_types=score_types,
+        target=target
+    )
+    demisto.debug("Response received: {0}".format(response))
+    alert_id = response.get("id")
+
+    markdown = "Alert **{0}** created".format(alert_id)
+
+    results = CommandResults(
+        readable_output=markdown,
+        outputs_prefix="SecurityScorecard.ScoreThresholdAlert.id",
+        outputs=alert_id
+    )
+
+    return results
+
+
+def securityscorecard_alert_delete_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    """`securityscorecard_alert_delete_command`: Delete an alert
+    See https://securityscorecard.readme.io/reference#delete_users-by-username-username-alerts-grade-alert
+    See https://securityscorecard.readme.io/reference#delete_users-by-username-username-alerts-score-alert
+
+    :type ``client``: ``Client``
+    :type `` args``: ``Dict[str, Any]``
+        ``args['alert_id']``: Alert ID, type ``String``
+        ``args['alert_type']``: Alert type. Can be either `score` or `grade`, type ``String``
+
+    :return:
+        A ``CommandResults`` object that is passed to ``return_results``
+    :rtype: ``CommandResults``
+    """
+
+    email = demisto.params().get('username').get("identifier")
+    alert_id = str(args.get("alert_id"))
+    alert_type = str(args.get("alert_type"))
```

Comment:
Why do we need type to delete?

Reply:
Their API has two separate endpoints for each type of alert: 
```
DELETE /users/by-username/username/alerts/grade/alert
DELETE /users/by-username/username/alerts/score/alert
```

So I decided, instead of having two different commands, to create one command where the user will specify whether this is a `grade` or `score` alert that they want to delete. WDYT?